app/repositories/ApprovalRepository.scala

```diff
@@ -115,7 +117,9 @@ class ApprovalRepositoryImpl @Inject() (implicit mongoComponent: ReactiveMongoCo
 
     logger.info(s"updating status of process $id to ${statusInfo.status} to collection $collectionName")
     val selector = Json.obj("_id" -> id)
-    val modifier = Json.obj("$inc" -> Json.obj("version" -> 1), "$set" -> Json.obj("meta.status" -> statusInfo.status))
+    val modifier = Json.obj(
+      "$inc" -> Json.obj("version" -> 1),
```

Comment:
Do we really want to change the version number here. I would just update the status on the latest version of the process.

Reply:
good point, probably better not to update version